Multisig/UI/UI Library/LoadableView.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  LoadableView.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 24.06.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+protocol LoadableViewModel: ObservableObject {
+    var isLoading: Bool { get set }
+    var errorMessage: String? { get set }
+}
+
+protocol LoadableView: View {
+    associatedtype LoadableModel: LoadableViewModel
+    var model: LoadableModel { get }
+}
+
+struct Loadable<V: LoadableView>: View {
```

Comment:
I thought about it too. But `Loadable(SomeView())` looks organic for me as. I think it is better than `LoadableView(SomeView())`
But I don't have a strong opinion here. @DmitryBespalov wdyt?

Reply:
I'd go with Mouaz's suggestion here, because also will be more consistent with other classes we have

Reply:
So what would be your suggestion here?
`LoadableViewModel `
`Loadable` (former LoadableView)
`LoadableView` (former Loadable)

I think it will be more confusing. Or did I get it wrong @MouazAlzahabi ?

Reply:
yes, like that

Reply:
and rename the "V" to the "Content" - also "view" variable can be named "content"

Reply:
all done